server/src/ArmZabbixAPI.cc

```diff
@@ -172,6 +174,21 @@ gpointer ArmZabbixAPI::mainThread(HatoholThreadArg *arg)
 	return ArmBase::mainThread(arg);
 }
 
+void ArmZabbixAPI::makeHatoholALLTriggers(void)
+{
+	ThreadLocalDBCache cache;
+	TriggerInfoList mergedTriggerInfoList;
+	ItemTablePtr triggers, expanded, mergedTriggers;
```

Comment:
This is not a bug.
However, in general, objects should be declared just before it's really used
to delay the execution of the constructor.